azure-devops/azext_devops/dev/common/pip_helper.py

```diff
@@ -17,4 +17,7 @@ def install_keyring():
 def _install_package(package_name):
     logger.debug('installing %s', package_name)
     pip_args = ['install', package_name]
-    _run_pip(pip_args)  # pylint: disable=protected-access
+    pip_status_code = _run_pip(pip_args)  # pylint: disable=protected-access
+    if pip_status_code > 0:
+        raise CLIError('An error occurred. Pip failed with status code {}. '
+                       'Use --debug for more information.'.format(pip_status_code))
```

Comment:
Minor: We can add package_name in message it self.

Reply:
done